docs/command/mongocli-ops-manager-automation-watch.txt

```diff
@@ -14,6 +14,11 @@ mongocli ops-manager automation watch
 
 Watch for automation changes to be completed.
 
+This command checks the automation goal version periodically until it's reached.
```

Comment:
Same as above? What is automation goal version?

Reply:
goal version comes from this api https://docs.opsmanager.mongodb.com/current/reference/api/automation-status/

Reply:
Yes agreed

Reply:
Proposing: This command checks the automation "lastGoalVersionAchieved" periodically until it matches "goalVersion".